.cspell.json

```diff
@@ -60,6 +61,8 @@
         "\\]\\(#.+?\\)",
         "^\\| GPU  Name.+$",
         "^gdown .+$",
+        "inet6 .*",
```

Comment:
I couldn't find any error with this word. I guess it's because all the words are written in the `console` block.

```suggestion
```

Reply:
Yes, the error is mentioned in the `console` block, and it detects ipv6 address. So I think the row including "inet6" should be ignored. Is this incorrect way?

![Screen Shot 2022-07-07 at 20 39 22](https://user-images.githubusercontent.com/89968601/177764876-cd23ae14-1846-400f-ab6a-8df42a2b2ea4.png)

Reply:
Ah, I'm sorry I misunderstood. What you wanted to ignore was not `inet6` but the words after `inet6`.

Reply:
How about this? I believe it should be more specific.

```suggestion
        "inet6 [a-z0-9:]*/64",
```